unison-cli/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -1437,6 +1437,12 @@ handleFindI isVerbose fscope ws input = do
       -- results may contain things in lib.
       pped <- Cli.currentPrettyPrintEnvDecl
       pure (pped, names)
+    FindLocalSubnamespace p -> do
```

Comment:
Suggestion: just add the `p` to `FindLocal` since the empty path is like `find-in <right-here>`